aspnet/publishing/azure-continuous-deployment.rst

```diff
@@ -1,8 +1,246 @@
+
+4. In the **Web App** blade, enter a unique value for the **App Service Name**. 
+
+
+	.. image:: azure-continuous-deployment/_static/06-azure-newappblade.png
+	
+	|
+	
+	.. note:: The **App Service Name** name needs to be unique. The portal will enforce this rule when you attempt to enter the name. After you enter a different value, you'll need to substitute that value for each occurrence of **SampleWebAppDemo** that you see in this tutorial. 
+
+	Also in the **Web App** blade, select an existing **App Service Plan/Location** or create a new one. If you create a new plan, select the pricing tier, location, and other options. For more information on App Service plans, `Azure App Service plans in-depth overview <https://azure.microsoft.com/en-us/documentation/articles/azure-web-sites-web-hosting-plans-in-depth-overview/>`_.
+
+5. Click **Create**. Azure will provision and start your web app.
+
+.. image:: azure-continuous-deployment/_static/07-azure-webappblade.png
+
+Enable Git publishing for the new web app
+-----------------------------------------
+
+Git is a distributed version control system that you can use to deploy your Azure App Service web app. You'll store the code you write for your web app in a local Git repository, and you'll deploy your code to Azure by pushing to a remote repository.   
+
+1. Log into the `Azure Portal <https://portal.azure.com>`_, if you're not already logged in.
+
+2. Click **Browse**, located at the bottom of the navigation pane.
+
+3. Click **Web Apps** to view a list of the web apps associated with your Azure subscription.
+
+4. Select the web app you created in the previous section of this tutorial.
+
+5. If the **Settings** blade is not shown, select **Settings** in the **Web App** blade. 
+
+6. In the **Settings** blade, select **Continuous deployment** > **Choose Source** > **Local Git Repository**. 
+
+	.. image:: azure-continuous-deployment/_static/08-azure-localrepository.png
+
+7. Click **OK**.
+
+8. If you have not previously set up deployment credentials for publishing a web app or other App Service app, set them up now:
+
+	* Click **Settings** > **Deployment credentials**. The **Set deployment credentials** blade will be displayed.
+
+	* Create a user name and password.  You'll need this password later when setting up Git.
+
+	* Click **Save**.
+
+9. In the **Web App** blade, click **Settings** > **Properties**. The URL of the remote Git repository that you'll deploy to is shown under **GIT URL**.
+
+10. Copy the **GIT URL** value for later use in the tutorial.
+
+	.. image:: azure-continuous-deployment/_static/09-azure-giturl.png
+
+Publish your web app to Azure App Service
+-----------------------------------------
+
+In this section, you will create a local Git repository using Visual Studio and push from that repository to Azure to deploy your web app. The steps involved include the following:
+
+	* Add the remote git reference to deploy your local repository to Azure.
```

Comment:
> Add a remote git reference

In Git I believe the common terminology is a "remote repository" or just a "remote" for short. See http://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes.